db_schema/migrations/10_anchor_episodes_page.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS anchorEpisodesPage (
+  account_id INTEGER NOT NULL,
+  episode_id VARCHAR(128) NOT NULL,
+  web_episode_id VARCHAR(128) NOT NULL,
+  PRIMARY KEY (account_id, episode_id, web_episode_id)
+);
+
+INSERT INTO migrations (migration_id, migration_name) VALUES (10, 'anchor episodesPage');
```

Comment:
has to be increased to 11 now due to other PR that introduced 10.

Reply:
done